docs/docfx.json

```diff
@@ -57,6 +57,7 @@
         "NuGet"
       ],
       "uhfHeaderId": []
+        "MSDocsHeader-DotNet"
```

Comment:
@JonDouglas Is that the format used in .NET docs? For VS, here is the format:
     "uhfHeaderId": "MSDocsHeader-VisualStudio",
so I don't see an array